### DISCUSSION TITLE
create subsscriptions - connection failure

### DISCUSSION THREAD

**Discussion by Rammurthy5** [pgEdge/cli#14] [Q&A]
Hello, 
I have ubuntu 22.04 EC2 running, stuck at subscriptions creation. 
I run the following command 
`./nodectl spock sub-create sub_n1n2 'host=<privateIP of node2> port=5432 user=root dbname=demo' demo`

i used `root` for user in the above command because thats got superuser privileges in node2. i have tried changing to `admin` which DB username, then tried supplying --password with value still the same error. what am i doing wrong?

gets the error 

```
ERROR: could not connect to the postgresql server: connection to server at "privateIP of node2", port 5432 failed: fe_sendauth: no password supplied
ERROR:
```
both the nodes are set without key-pair, same vpc, subnet.

**Comment by cmotyka** [comment]
As per the quick start, to use pgEdge platform you should use a non-root user with passwordless sudo privileges

**Reply by Rammurthy5** [reply]
Should I change an existing user to have root privileges and then try executing the command again? Do I understand it correctly ?

**Reply by cmotyka** [reply]
The user has to be not root and have passwordless sudo set up (https://www.ibm.com/docs/en/zscc/1.1.0?topic=eylzsdg-enable-passwordless-sudo-access-your-linux-user-id)
So everything from the command `./nodectl install pgedge -U superuser-name -P superuser-password -d database-name` and on should be executed by a non-root user.

**Reply by Rammurthy5** [reply]
Superuser-name and superuser-password mentioned in the following command - does it mean the root or any user of the EC2 that has root privileges ? 

./nodectl install pgedge -U superuser-name -P superuser-password -d database-name
